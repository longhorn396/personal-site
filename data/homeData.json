{
  "projects": [
    {
      "githubLink": "",
      "name": "Autoscraper",
      "projectLink": "https://autoscraper-ui.herokuapp.com/",
      "summary": "A tool to learn about the depreciation of different cars over time and the automobile market as a whole."
    }, {
      "githubLink": "https://github.com/longhorn396/code-golf",
      "name": "Code Golf",
      "projectLink": "",
      "summary": "Simple coding puzzles written in multiple languages, multiple patterns, and comparison between the patterns."
    }, {
      "githubLink": "https://github.com/longhorn396/personal-site",
      "name": "Personal Website",
      "projectLink": "/",
      "summary": "The website you're currently visiting written with Next.js for React and Material-UI hosted in AWS."
    }, {
      "githubLink": "https://github.com/longhorn396/WikIRMan",
      "name": "WikIRMan",
      "projectLink": "",
      "summary": "A game of Hangman where the words are pulled from a random Wikipedia article that is shown to the player."
    }
  ],
  "sections": [
    {
      "to": "about",
      "display": "About"
    },
    {
      "to": "work",
      "display": "Experience"
    },
    {
      "to": "projects",
      "display": "Projects"
    },
    {
      "to": "contact",
      "display": "Contact"
    }
  ],
  "work": [
    {
      "name": "Gartner",
      "summary": [
        "My first six month rotation was on the Cloud Automation team where I fell in love with working with cloud technologies. I used Terraform to deploy Lambda functions that enforced policies such as S3 disaster recovery replication and proper logging for load balancers, CDNs, and firewalls. I also created a command line utility that simplified running boto3 scripts across multiple accounts.",
        "I then spent rotations on a frontend development team, the Performance Engineering team, and a fullstack development team. While I learned a lot (including many of the libraries and concepts of this website), I always wanted to come back to the Cloud. Now, I am a member of our EKS team as more and more of our developers look to onboard onto Kubernetes."
      ]
    }, {
      "name": "vAuto: Cox Automotive",
      "summary": [
        "The VIN Diesel team was working on a company-wide initiative that left their backlog all but ignored. After showing my strengths and work ethic, I was allowed to take just about any ticket from that backlog and run with it, something I love doing. I not only got to work with .NET and C#, but also learned just how important elements such as company culture and team camaraderie were to my success."
      ]
    }, {
      "name": "CDK Global",
      "summary": [
        "The Data Platform team was my first Agile sprint team, which meant I was learning a new process of working in addition to completely new technologies such as Avro, Kafka, and Hadoop. By the end of the internship, my teammate and I had added several features to a supporting application that increased the user experience of the platform as a whole."
      ]
    }, {
      "name": "The University of Texas Health Science Center at Houston (UTHealth)",
      "summary": [
        "As a summer intern, I was brought onto the Academic Technology team to help update and migrate a catalog of Tomcat apps to new servers. Once I got comfortable with the technologies in their tech stack, I gained more responsibilities and ultimately earned a co-op position as a web developer."
      ]
    }
  ]
}