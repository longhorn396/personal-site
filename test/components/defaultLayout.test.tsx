/**
 * @jest-environment jsdom
 */
import { shallow } from 'enzyme';
import React from 'react';
import homeData from '../../data/homeData.json';
import Default from '../../components/DefaultLayout';

const { sections } = homeData;
const layout = shallow(<Default description='' sections={sections} title=''></Default>);
 
describe('Default layout', () => {
  it('matches snapshot', () => {
    expect(layout).toMatchSnapshot();
  });
});
 