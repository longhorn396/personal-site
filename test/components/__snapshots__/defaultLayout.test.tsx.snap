// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default layout matches snapshot 1`] = `
<ThemeProvider
  theme={
    Object {
      "breakpoints": Object {
        "between": [Function],
        "down": [Function],
        "keys": Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ],
        "only": [Function],
        "up": [Function],
        "values": Object {
          "lg": 1280,
          "md": 960,
          "sm": 600,
          "xl": 1920,
          "xs": 0,
        },
        "width": [Function],
      },
      "direction": "ltr",
      "mixins": Object {
        "gutters": [Function],
        "toolbar": Object {
          "@media (min-width:0px) and (orientation: landscape)": Object {
            "minHeight": 48,
          },
          "@media (min-width:600px)": Object {
            "minHeight": 64,
          },
          "minHeight": 56,
        },
      },
      "overrides": Object {
        "MuiAccordion": Object {
          "root": Object {
            "&$expanded": Object {
              "margin": "1px 0",
            },
          },
        },
      },
      "palette": Object {
        "action": Object {
          "activatedOpacity": 0.24,
          "active": "#fff",
          "disabled": "rgba(255, 255, 255, 0.3)",
          "disabledBackground": "rgba(255, 255, 255, 0.12)",
          "disabledOpacity": 0.38,
          "focus": "rgba(255, 255, 255, 0.12)",
          "focusOpacity": 0.12,
          "hover": "rgba(255, 255, 255, 0.08)",
          "hoverOpacity": 0.08,
          "selected": "rgba(255, 255, 255, 0.16)",
          "selectedOpacity": 0.16,
        },
        "augmentColor": [Function],
        "background": Object {
          "default": "#0a192f",
          "paper": "#333f58",
        },
        "common": Object {
          "black": "#000",
          "white": "#fff",
        },
        "contrastThreshold": 3,
        "divider": "rgba(255, 255, 255, 0.12)",
        "error": Object {
          "contrastText": "#fff",
          "dark": "#d32f2f",
          "light": "#e57373",
          "main": "#f44336",
        },
        "getContrastText": [Function],
        "grey": Object {
          "100": "#f5f5f5",
          "200": "#eeeeee",
          "300": "#e0e0e0",
          "400": "#bdbdbd",
          "50": "#fafafa",
          "500": "#9e9e9e",
          "600": "#757575",
          "700": "#616161",
          "800": "#424242",
          "900": "#212121",
          "A100": "#d5d5d5",
          "A200": "#aaaaaa",
          "A400": "#303030",
          "A700": "#616161",
        },
        "info": Object {
          "contrastText": "rgba(0, 0, 0, 0.87)",
          "dark": "#14cba8",
          "light": "#9efff",
          "main": "#64ffda",
        },
        "primary": Object {
          "contrastText": "#fff",
          "dark": "#000007",
          "light": "#333f58",
          "main": "#0a192f",
        },
        "secondary": Object {
          "contrastText": "#fff",
          "dark": "#870000",
          "light": "#f9683a",
          "main": "#bf360c",
        },
        "success": Object {
          "contrastText": "rgba(0, 0, 0, 0.87)",
          "dark": "#388e3c",
          "light": "#81c784",
          "main": "#4caf50",
        },
        "text": Object {
          "disabled": "rgba(255, 255, 255, 0.5)",
          "hint": "rgba(255, 255, 255, 0.5)",
          "icon": "rgba(255, 255, 255, 0.5)",
          "primary": "#ffffff",
          "secondary": "#ffffff",
        },
        "tonalOffset": 0.2,
        "type": "dark",
        "warning": Object {
          "contrastText": "rgba(0, 0, 0, 0.87)",
          "dark": "#f57c00",
          "light": "#ffb74d",
          "main": "#ff9800",
        },
      },
      "props": Object {},
      "shadows": Array [
        "none",
        "0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12)",
        "0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12)",
        "0px 3px 3px -2px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 1px 8px 0px rgba(0,0,0,0.12)",
        "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
        "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
        "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
        "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
        "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
        "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
        "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
        "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
        "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
        "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
        "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
        "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
        "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
        "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
        "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
        "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
        "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
        "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
        "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
        "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
        "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
      ],
      "shape": Object {
        "borderRadius": 4,
      },
      "spacing": [Function],
      "transitions": Object {
        "create": [Function],
        "duration": Object {
          "complex": 375,
          "enteringScreen": 225,
          "leavingScreen": 195,
          "short": 250,
          "shorter": 200,
          "shortest": 150,
          "standard": 300,
        },
        "easing": Object {
          "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
          "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
          "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
          "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
        },
        "getAutoHeightDuration": [Function],
      },
      "typography": Object {
        "body1": Object {
          "fontFamily": "Cambria, Cochin, Georgia, Times, Times New Roman, serif",
          "fontSize": "1.1428571428571428rem",
          "fontWeight": 400,
          "lineHeight": 1.5,
          "marginBottom": "0.5rem",
        },
        "body2": Object {
          "fontFamily": "Cambria, Cochin, Georgia, Times, Times New Roman, serif",
          "fontSize": "1rem",
          "fontWeight": 400,
          "lineHeight": 1.43,
          "marginBottom": "0.5rem",
        },
        "button": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "1rem",
          "fontWeight": 500,
          "lineHeight": 1.75,
          "textTransform": "uppercase",
        },
        "caption": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "0.8571428571428571rem",
          "fontWeight": 300,
          "lineHeight": 1.66,
        },
        "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
        "fontSize": 16,
        "fontWeightBold": 700,
        "fontWeightLight": 300,
        "fontWeightMedium": 500,
        "fontWeightRegular": 300,
        "h1": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "6.857142857142857rem",
          "fontWeight": 300,
          "lineHeight": 1.167,
        },
        "h2": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "4.285714285714286rem",
          "fontWeight": 300,
          "lineHeight": 1.2,
        },
        "h3": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "3.4285714285714284rem",
          "fontWeight": 300,
          "lineHeight": 1.167,
        },
        "h4": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "2.4285714285714284rem",
          "fontWeight": 300,
          "lineHeight": 1.235,
        },
        "h5": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "1.7142857142857142rem",
          "fontWeight": 300,
          "lineHeight": 1.334,
        },
        "h6": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "1.4285714285714284rem",
          "fontWeight": 500,
          "lineHeight": 1.6,
        },
        "htmlFontSize": 16,
        "overline": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "0.8571428571428571rem",
          "fontWeight": 300,
          "lineHeight": 2.66,
          "textTransform": "uppercase",
        },
        "pxToRem": [Function],
        "round": [Function],
        "subtitle1": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "1.1428571428571428rem",
          "fontWeight": 300,
          "lineHeight": 1.75,
        },
        "subtitle2": Object {
          "fontFamily": "Lucida Sans,Lucida Sans Regular,Geneva,Verdana,sans-serif",
          "fontSize": "1rem",
          "fontWeight": 500,
          "lineHeight": 1.57,
        },
      },
      "zIndex": Object {
        "appBar": 1100,
        "drawer": 1200,
        "mobileStepper": 1000,
        "modal": 1300,
        "snackbar": 1400,
        "speedDial": 1050,
        "tooltip": 1500,
      },
    }
  }
>
  <WithStyles(CssBaseline) />
  <style>
    
        body {
          margin: 0;
        }
        .center {
          justify-content: center;
          text-align: center;
        }
        .dark-info {
          color: #14cba8;
        }
        section {
          padding: 100px 0px;
        }
        section:last-child {
          padding-bottom: 25px;
        }
      
  </style>
  <Navbar
    sections={
      Array [
        Object {
          "display": "About",
          "to": "about",
        },
        Object {
          "display": "Experience",
          "to": "work",
        },
        Object {
          "display": "Projects",
          "to": "projects",
        },
        Object {
          "display": "Contact",
          "to": "contact",
        },
      ]
    }
  />
  <Head>
    <title>
      Devin Drawhorn
    </title>
    <meta
      content="Devin Drawhorn"
      name="description"
    />
    <meta
      content="/og.png"
      name="image"
    />
    <meta
      content="Devin Drawhorn"
      name="og:title"
    />
    <meta
      content="Devin Drawhorn"
      name="og:description"
    />
    <meta
      content="/og.png"
      name="og:image"
    />
    <meta
      content="https://drawhorn.click"
      name="og:url"
    />
    <meta
      content="website"
      name="og:type"
    />
    <meta
      content="summary_large_image"
      name="twitter:card"
    />
    <meta
      content="Devin Drawhorn"
      name="twitter:title"
    />
    <meta
      content="Devin Drawhorn"
      name="twitter:description"
    />
    <meta
      content="/android-chrome-512x512.png"
      name="twitter:image"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <meta
      charSet="utf-8"
    />
  </Head>
  <WithStyles(ForwardRef(Container))
    maxWidth="lg"
    role="main"
  />
</ThemeProvider>
`;
